game/src/test/scala/io/aigar/game/CellSpec.scala

```diff
@@ -63,5 +63,21 @@ class CellSpec extends FlatSpec with Matchers {
 
     cell.position should equal(new Vector2(42f, 42f))
   }
+
+  it should "update its behavior on update" in {
+    val cell = new Cell(1, new Vector2(0f, 0f))
+    cell.behavior = new SpyBehavior
+
+    cell.update(1f)
+
+    cell.behavior shouldBe 'updated
+  }
 }
 
+class SpyBehavior extends SteeringBehavior {
```

Comment:
In the sense of a [test spy](`https://en.wikipedia.org/wiki/Test_double`). I changed the name to just `TestBehavior` in a next PR! It's just to be able to check that `update` was called.